README.md

```diff
@@ -0,0 +1,75 @@
+# clay-sitemaps
+
+A suite a tools for generating sitemaps for Clay sites.
+
+Clay-sitemaps provides two "standard" sitemap middleware, one for text and one for XML, that can get you up and running quickly. It also provides utilities that can help you create your own sitemap logic.
+
+## Quick Start
+
+```
+const sitemaps = require('clay-sitemaps')(someAmphoraInstance);
+
+app
+  .get('/sitemap.txt', sitemaps.standardText())
+  .get('/sitemap.xml', sitemaps.standardXML({
+    engines: { // engines for rendering sitemap templates
+      handlebars: yourHandlebarsEngine
```

Comment:
Thoughts about switching to middleware model via **use** to hand off control and encapsulate the logic?  Moving whatever engine init up to the sitemap middleware "constructor".   for example, for google news sitemap, do we have to edit sites app.js to add route handler or instead just release an update of sitemaps middleware.